CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ Visit The National Archives' [documentation on OGL and Crown copyright](https://
 
 This was added in [pull request #2702: Allow localisation of content licence and copyright notices in Footer](https://github.com/alphagov/govuk-frontend/pull/2702).
 
+#### Pass HTML directly into compatible components
+
+If using the Nunjucks macros, you can now pass HTML content directly into compatible components using [the Nunjucks `call` syntax](https://mozilla.github.io/nunjucks/templating.html#call). In these instances, the called HTML is treated the same as the `html` parameter.
```

Comment:
Two thoughts:

1. I know this is very unlikely to be used by somebody who isn't familiar with Nunjucks, but should we make it clear how this interacts with `html` and `text` parameters? I could potentially see an edge case where somebody was passing in those options AND the called HTML and getting confused (but this does feel like an edge case!)
2. Should we add a little "this is in addition to the Fieldset component, which is already callable"? I can't see that note in the rest of the changelog, and might be useful for folks interested in this to see if they don't know about it.

Reply:
1. Yeah, that sounds good - no harm in being really clear I think.
2. Basically just using the shiny new things to help highlight the sorta shiny old thing...

Reply:
I've updated the changelog.

Reply:
I think you've formatted the entire CHANGELOG file in that latest push.

Reply:
🤦 Ack! I was fiddling with extensions last week and forgot to turn that one off. Fixed.